scripts/asamanager.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+# Note: Windows Bash doesn't support shebang extra params
+set -e
+
+if [[ "$DOCKER_TAG" == "" ]]; then
+    export DOCKER_TAG=testing
+fi
+
+DOCKER_IMAGE="azureiotpcs/asa-manager-dotnet:$DOCKER_TAG"
+DOCKER_PORT=9024
+DOCKER_NAME="asa-manager"
+DOCKER_NETWOK="integrationtests"
+
+APP_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )/"
+cd $APP_HOME
+
+source "$APP_HOME/scripts/.functions.sh"
+
+check_env_variables()
+{
+	variables_not_found=0
```

Comment:
indentation

Reply:
very weird when I look at it in visual studio/notepad++ indentation is normal